### PR TITLE
BIA-1170: Implement API to AMT parquet conversion - Assessment - Stud…

### PR DESCRIPTION
**Pull request by jegasape**
BIA-1170: Implement API to AMT parquet conversion - Assessment - StudentAssessmentFact.

### CODE REVIEW

File: src/edfi_amt_data_lake/parquet/amt/asmt/assessment_student/main.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    result_frame['AssessmentIdentifier'] = (
+        result_frame['assessmentReference.assessmentIdentifier']
+    )
+
+    result_frame['AssessmentIdentifier'] = (
+        result_frame['administrationDate']
+    )
+ 
+    if not os.path.exists(f"{config('PARQUET_FILES_LOCATION')}"):
+        os.makedirs(f"{config('PARQUET_FILES_LOCATION')}")
```
Comment (DavidJGapCR) [inline src/edfi_amt_data_lake/parquet/amt/asmt/assessment_student/main.py:137]: Yes, I think we need this code, but I am thinking we can add it to the pandas wrapper, save parquet function?

Comment (DavidJGapCR) [reply src/edfi_amt_data_lake/parquet/amt/asmt/assessment_student/main.py:137]: Actually we don't need this code. It's covered on [BIA-1215](https://tracker.ed-fi.org/browse/BIA-1215). Please just remove.

File: src/edfi_amt_data_lake/parquet/amt/asmt/assessment_student/main.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        recordPath=None,
+        meta=[],
+        metaPrefix=None,
+        recordPrefix=None,
+        errors='ignore'
+      )
+
+    student_assessemnt_score_result_frames_assessment_reporting_method_descriptor = jsonNormalize(
+        student_assessment,
+        recordPath=['scoreresult_frames'],
```
[the remark this answers is not in the document]
Comment (jegasape) [reply src/edfi_amt_data_lake/parquet/amt/asmt/assessment_student/main.py]: Done.

**Comment by DavidJGapCR** [comment]
I think you forgot to include the changes on `asmt_collection.py`

**Comment by DavidJGapCR** [comment]
This is looking so much better. But still there are things to review:

. Noticed that in the database the `StudentObjectiveAssessmentKey` and `ObjectiveAssessmentKey` are empty, but in the parquet file those rows have values. 

. You have a column with name `Assessment`. I think (not 100%) this column name should be `AssessmentKey.` 

. Something to review: If you see these fields in the amt view: `StudentScore, ResultDataType, ReportingMethod, PerformanceResult`. These fields are interesting because we try to find the value from the **ObjectiveAssessment** level, but if there is no value (we have left join) then we take it from the **AssessmentLevel**. If you review the design of the view you will see what I mean. If not, let me know. 
image

. We are missing the `PerformanceResult` values. This value should be taken from the `performanceLevelDescriptors` endpoint. 
image

. The parquet file has more rows: 17890. When there are 10168 in the database.

**Review by DavidJGapCR — APPROVED** [review]

**Pull request merged**